lntest/harness.go

```diff
@@ -281,6 +281,7 @@ func (n *NetworkHarness) ConnectNodes(ctx context.Context, a, b *HarnessNode) er
 			Pubkey: bobInfo.IdentityPubkey,
 			Host:   b.cfg.P2PAddr(),
 		},
+		Perm: true,
```

Comment:
Not sure if this should be set to perm, as there're conditions where we want to connect peers in the tests, but then after a restart they shouldn't attempt to automatically reconnect.

Reply:
Agreed here.

Reply:
I mainly added this to be sure this doesn't get us stuck in any weird states, the itests seem to handle it just fine. Will revert.